docs/chapters/baseplatform/building-PELUX-sources.rst

```diff
@@ -95,7 +94,7 @@ configuration samples from.
 
     TEMPLATECONF=`pwd`/sources/meta-pelux/conf/variant/intel source sources/poky/oe-init-build-env build
 
-The script will create configs if there are no configs present, a message about
+The script will create configurations if there are no configurations present. A message about
```

Comment:
An output on the logs regarding the created `conf/local.conf` ........

Reply:
In that case I think it should be "printout", since this command is not only run e.g. with a build tool like Jenkins. When I run this it's very often command-line.